zuul.d/jobs.yaml

```diff
@@ -463,3 +463,10 @@
     vars:
       container_command: podman
     nodeset: ubuntu-bionic-2vcpu
+
```

Comment:
This looks right, but lets move it into zuul.d/ansible-navigator-jobs.yaml

Reply:
You can also add the job to https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L54 which attaches it to the check / gate pipelines for the navigator project.

Reply:
thx paul